resources/js/views/Recipes.vue

```diff
@@ -21,6 +21,7 @@
       <template v-slot:item.action="{ item }">
         <v-icon small class="mr-2" @click="editItem(item)">mdi-pencil</v-icon>
         <v-icon small @click="deleteItem(item)">mdi-delete</v-icon>
+        <v-btn color="primary" class="ml-4" @click="editItem(item)">Download</v-btn>
```

Comment:
so maybe we cut the edit modal out of the mix and make this onclick download the selected item download(item) perhaps

Reply:
We should leave the edit model, its useful still

Reply:
Ahh I see what you mean, I like how its there though because it gives more customization for the user to select if they want to download and store as another category. Fried chicken can be for lunch or dinner so its open for them